lib/RepositorySettings.js

```diff
@@ -47,6 +47,11 @@ function RepositorySettings(options) {
      */
     this.contributorsPath = options.contributorsPath;
 
+    /**
+     * Use the GitHub Contributors API
+     */
+    this.contributorsFromGitHub = defaultValue(options.contributorsFromGitHub, false);
```

Comment:
`README.md` will need to be updated with this option. Sorry I didn't notice that in my initial review.